configuration.rst

```diff
@@ -445,6 +445,32 @@ In order to define the actual values of env vars, Symfony proposes different
 solutions depending if the application is running in production or in your local
 development machine.
 
+Independent from the way you set environmnet variables, you may need to run the
+``debug:container`` command with the ``--env-vars`` option to verify that the
+env vars are defined and have the expected values:
```

Comment:
Previously „environment variables“ is used, here „env vars“. Please use only one version for consistency.

Reply:
I've reworded this a bit ... but in the article we mix "environment variables" and "env vars". At the beginning we say "environment variables" to not lose newcomers and we tell them that "env vars" is the same. Later we mostly use "env vars" because "environment variables" is long and boring.